os/drivers/video/Kconfig

```diff
@@ -29,4 +29,90 @@ config VIDEO_SOURCE
 config VIDEO_NULL
 	bool "Driver for Dummy Video lowerhalf"
 	depends on VIDEO_SOURCE
-	default n
\ No newline at end of file
+	default n
+
+if VIDEO_NULL
```

Comment:
In my opinion below are belong to null driver, so prefix is needed. 
How do you think?

Reply:
Agreed, done.